sechub-pds-solutions/multi/docker/Multi-Debian.dockerfile

```diff
@@ -39,6 +39,11 @@ COPY packages.txt $TOOL_FOLDER/packages.txt
 # Interesting idea, but not as useful inside a container, which in essence is already a virtual environment.
 # Use `--break-system-packages` to let the Python package manager `pip` mix packages from Debian and Python
 RUN pip install --break-system-packages -r $TOOL_FOLDER/packages.txt
+# The pip -e option seems to be broken in recent versions of setuptools
+# when it comes to installing modules from the local filesystem,
+# see: https://github.com/pypa/setuptools/issues/3301
+# this is why we download the plugin using a GIT http call and not use GIT submodules
+RUN pip install --break-system-packages -e "git+https://github.com/alexdd/bandit-sarif-formatter#egg=bandit-sarif-formatter"
```

Comment:
What does `-e` stand for? Is there a long form for `-e`? 
I would prefer to fix the git URL to a specific commit/tag, is that possible?

Reply:
Both is possible I updated the pull request. I won't put any more commits to the `alexdd/bandit-sarif-formatte`r repo, so `@main` should be fine (just for example).